tests/test_binaries.py

```diff
@@ -120,6 +120,25 @@ def testArgumentsAssertions(self):
         expected = self._loadVisitedSet(os.path.join(dirname, 'reference', 'arguments_linux_amd64_visited.txt'))
         self.assertGreaterEqual(actual, expected)
 
+    def testArgumentsAssertionsArmv7(self):
+        dirname = os.path.dirname(__file__)
```

Comment:
Can we move this to a global variable e.g. `DIR_PATH = ...` ? some other tests might want to use that too :P

Reply:
E.g. the next test also uses this line:
```
     def testDecree(self):
         dirname = os.path.dirname(__file__)```

Reply:
ok